modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -44,7 +44,9 @@ sda.enkf.multisite <- function(settings,
                                             debug = FALSE,
                                             pause = FALSE,
                                             Profiling = FALSE,
-                                            OutlierDetection=FALSE),
+                                            OutlierDetection=FALSE,
+                                            free_run = FALSE,
```

Comment:
If you're adding new variables, make sure to update both the ROxygen and the changelog

Reply:
Also, it doesn't look like either of these variables are actually used. Instead they are being accessed from within the `control` object. But they still need to be documented